docs/manual-setup.adoc

```diff
@@ -224,7 +202,7 @@ in one of your TDT IDs (mine was 0x530D37eD3E2194EC1671DD2E153A03FCED5F80cD).
 
 Generate a bitcoin wallet address by running:
 ```
-bitcoind-wallet getNewAddress
+$ bitcoind-wallet getNewAddress
```

Comment:
At some point we should probably go ahead and make this change throughout the file, but for now let's roll with what we've got 🚢